doc/source/cluster/vms/user-guides/launching-clusters/ibm-gen2.md

```diff
@@ -0,0 +1,79 @@
+
+# Launching Ray Clusters on IBM VPC
+
+This guide details the steps needed to start a Ray cluster on IBM VPC.
+
+To start a IBM VPC Ray cluster, you will use the Ray cluster launcher and
+interactive config tool to generate ray cluster config file for IBM VPC.
+
+
+```{note}
+The IBM VPC integration is community-maintained. Please reach out to the integration maintainers on Github
+if you run into any problems: kpavel.
+```
+
+## Using Ray cluster launcher
+
+
+### Install Ray cluster launcher
+
+The Ray cluster launcher is part of the `ray` CLI. Use the CLI to start, stop and attach to a running ray cluster using commands such as  `ray up`, `ray down` and `ray attach`. You can use pip to install the ray CLI with cluster launcher support. Follow [the Ray installation documentation](installation) for more detailed instructions.
+
+```bash
+# install ray
+pip install -U ray[default]
+```
+
+### Install ibm cloud client packages and config tool
+
+```bash
+# install dependencies
+pip install ibm_vpc ibm_platform_services ibm_cloud_sdk_core
+# install config tool
+pip install -U lithopscloud
```

Comment:
why `-U` here, but not in the previous command line? maybe note something in the comments

Reply:
Removed -U